src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20230224161839_CleanUpKeyStatData.sql

```diff
@@ -0,0 +1,9 @@
+﻿UPDATE ContentBlock
+SET ContentSectionId = NULL
+WHERE Type = 'DataBlock'
+  AND Id IN (SELECT DISTINCT DataBlockId FROM KeyStatisticsDataBlock)
+  AND ContentSectionId IS NOT NULL
+  AND ContentSectionId NOT IN (SELECT CS.Id FROM ContentSections CS WHERE CS.Type != 'KeyStatistics'); -- this condition for safety
+
+DELETE ContentSections FROM ContentSections WHERE Type = 'KeyStatistics'
```

Comment:
`DELETE FROM ContentSections` would be the way to write a regular delete statement.

`DELETE ContentSections FROM ContentSections` would only be needed as a result of joining with another table.